src/components/Header/components/Mobile.tsx

```diff
@@ -10,22 +10,20 @@ import SearchButton from "./SearchButton";
 const Mobile = chakra(
   ({ isLoginOpen, patronName, setIsLoginOpen }: LoginProps) => (
     <Flex>
-      <HStack>
-        <LoginButton
-          isMobile
-          isLoginOpen={isLoginOpen}
-          patronName={patronName}
-          setIsLoginOpen={setIsLoginOpen}
-        />
-        <Icon
-          id="locator-icon"
-          name="actionHelpOutline"
-          size="medium"
-          title="NYPL Locator"
-        />
-        <SearchButton isMobile />
-        <MobileNavButton />
-      </HStack>
+      <LoginButton
+        isMobile
+        isLoginOpen={isLoginOpen}
+        patronName={patronName}
+        setIsLoginOpen={setIsLoginOpen}
+      />
+      <Icon
```

Comment:
Are we waiting to do something with this NYPL Locator icon?

![Screen Shot 2022-06-22 at 2 06 19 PM](https://user-images.githubusercontent.com/57868332/175107252-0c23a2b0-ea75-4dc5-a4ce-2f3aa2a91efd.png)

Reply:
Yeah. It was recently added as an icon and now added in #1061